programs/clearing_house/src/controller/insurance.rs

```diff
@@ -41,18 +41,18 @@ pub fn add_insurance_fund_stake(
         "Insurance Fund balance should be non-zero for new LPs to enter"
     )?;
 
-    let if_shares_before = insurance_fund_stake.if_shares;
-    let total_if_shares_before = spot_market.total_if_shares;
-    let user_if_shares_before = spot_market.user_if_shares;
-
     apply_rebase_to_insurance_fund(insurance_vault_amount, spot_market)?;
     apply_rebase_to_insurance_fund_stake(insurance_fund_stake, user_stats, spot_market)?;
 
+    let if_shares_before = insurance_fund_stake.checked_if_shares(spot_market)?;
```

Comment:
The value of `*_before` changes as a result of this refactor, is this intended? Now the "before" values are calculated as prior to any rebasing

Reply:
hmmm i moved this after so i didnt need to call `unchecked_if_shares`, what do you prefer @0xbigz

Reply:
think after makes sense too, its just for record keeping either way